src/h2olog/h2olog.cc

```diff
@@ -63,6 +64,9 @@ Optional arguments:
   -a                Include application data which are omitted by default.
   -r                Run without dropping root privilege.
   -w <path>         Path to write the output (default: stdout).
+  -D <flag>         Enables a BCC debug flag (supported flag: DEBUG_LLVM_IR,
+                    DEBUG_BPF, DEBUG_PREPROCESSOR, DEBUG_SOURCE,
+                    DEBUG_BPF_REGISTER_STATE, DEBUG_BTF)
```

Comment:
Is the scope of this option limited to setting debug flags?

If true, then I do not think we need the "DEBUG_" prefix for each option. Otherwise, I would believe that this option is something to be used for setting an arbitrary flag (rather than those limited to debugging).

Reply:
That's a good question.

I rather think it could have any flags in the future. For example, we might migrate the BPF framework from BCC to libbpf + BPF CO-RE in the future, which could have flags that are not related to debugging. However, I don't have a plan to add other kinds of flags so far, so I describe it as "a BCC debug flag". 

Separately, these names directly come from BCC: https://github.com/iovisor/bcc/blob/master/src/cc/bpf_module.h#L43-L56 so I don't want to rename them anyway.

Reply:
Thank you for the clarification. I think the description is fine.

Though I think we might want to change the option name from "-D" to "-f" or something, assuming that we agree that the capability of the flag is not necessarily limited to debugging (each option _values_ represent them as debug flags, by having "DEBUG_" prefix).

Reply:
Agreed! Renamed it from `-D` to `-f`: 23f405932